src/SearchFilters.js

```diff
@@ -35,29 +35,53 @@ export let FiltersBox = withTheme(({ theme: { Box }, ...props }) => (
 ))
 FiltersBox.displayName = 'FiltersBox'
 
-let BasicSearchFilters = ({ setMode, trees, children, BasicFilters }) => (
-  <div>
-    <Flex alignItems="center" justifyContent="space-between">
-      <Flex alignItems="center">
-        <h1>Filters</h1>
-        <ToggleFiltersButton onClick={() => setMode('resultsOnly')} />
-      </Flex>
-      <div>
-        <TreePauseButton children={children} />
-      </div>
-    </Flex>
-    <LabelledList list={trees} Component={BasicFilters} />
-    <LinkButton onClick={() => setMode('builder')} style={{ marginTop: 15 }}>
-      Switch to Advanced Search Builder
-    </LinkButton>
+let PopOverItem = ({ children, ...props }) => (
+  <div style={{ margin: 10, cursor: 'pointer' }} {...props}>
+    {children}
   </div>
 )
 
+let BasicSearchFilters = ({ setMode, trees, children, BasicFilters }) => {
+  let [isOpen, setIsOpen] = React.useState(false)
+  return (
+    <div>
+      <Flex alignItems="center" justifyContent="space-between">
+        <Flex alignItems="center">
+          <h1>Filters</h1>
+        </Flex>
+        <div>
+          <Popover
+            isOpen={isOpen}
+            onClose={() => setIsOpen(false)}
+            style={{ width: 180, right: 0, top: 40 }}
+          >
+            <PopOverItem onClick={() => setMode('resultsOnly')}>
+              Toggle Filters
+            </PopOverItem>
+            <TreePauseButton children={children} Component={PopOverItem} />
+            <PopOverItem onClick={() => setMode('builder')}>
+              Advanced Search Builder
+            </PopOverItem>
+          </Popover>
+          <Icon
+            style={{ paddingTop: 15, cursor: 'pointer' }}
+            icon="more_vert"
+            onClick={() => setIsOpen(true)}
+          />
+        </div>
+      </Flex>
+      <LabelledList list={trees} Component={BasicFilters} />
+    </div>
+  )
+}
+
 let BuilderSearchFilters = ({ setMode, trees, BuilderFilters }) => (
   <div>
-    <Flex style={{ alignItems: 'center' }}>
-      <h1>Filters</h1>
-      <LinkButton onClick={() => setMode('basic')}>
+    <Flex justifyContent="space-between" style={{ alignItems: 'center' }}>
+      <Flex alignItems="center">
+        <h1>Filters</h1>
+      </Flex>
+      <LinkButton onClick={() => setMode('basic')} style={{ marginRight: 30 }}>
```

Comment:
It looks weird and off but I did forget that this is a repo where we should not add styling etc. Since it is something that most of the users would never see will remove the margin etc.

Before:
![image](https://user-images.githubusercontent.com/910303/84941107-ba463380-b095-11ea-9ae6-865b2ab355bb.png)

After:
![image](https://user-images.githubusercontent.com/910303/84940695-0d6bb680-b095-11ea-9e8f-c16202b177e4.png)